java/client/src/main/java/glide/api/RedisClient.java

```diff
@@ -78,4 +87,118 @@ public CompletableFuture<Object> customCommand(String[] args) {
                 Command.builder().requestType(Command.RequestType.CUSTOM_COMMAND).arguments(args).build();
         return commandManager.submitNewCommand(command, BaseClient::handleObjectResponse);
     }
+
+    /**
+     * Ping the Redis server.
+     *
+     * @see <a href="https://redis.io/commands/ping/">redis.io</a> for details.
+     * @returns the String "PONG"
+     */
+    @Override
+    public CompletableFuture<String> ping() {
+        Command command = Command.builder().requestType(Command.RequestType.PING).build();
+        return commandManager.submitNewCommand(command, BaseClient::handleStringResponse);
+    }
+
+    /**
+     * Ping the Redis server.
+     *
+     * @see <a href="https://redis.io/commands/ping/">redis.io</a> for details.
+     * @param msg - the ping argument that will be returned.
+     * @returns return a copy of the argument.
```

Comment:
1. Don't put dashes (https://github.com/Bit-Quill/glide-for-redis/pull/68#discussion_r1468113307)
2. Keep the casing consistent
```suggestion
     * @param msg The ping argument that will be returned.
     * @returns A copy of the argument.
```